ScoutSuite/providers/gcp/rules/findings/cloudsql-instances-public-ips.json

```diff
@@ -0,0 +1,29 @@
+{
+    "description": "Cloud SQL Database Instances Have Public IPs",
+    "rationale": "To lower the organization's attack surface, Cloud SQL databases should not have public IPs. Private IPs provide improved network security and lower latency for your application.",
+    "remediation": "From console:<ol><li>Go to the Cloud SQL Instances page in the Google Cloud Console by visiting https://console.cloud.google.com/sql/instances.</li><li>Click the instance name to open its Instance details page.</li><li>Select the <samp>Connections</samp> tab.</li><li>Deselect the <samp>Public IP</samp> checkbox.</li><li>Click <samp>Save</samp> to update the instance.</li></ol>",
+    "compliance": [
+        {
+            "name": "CIS Google Cloud Platform Foundations",
+            "version": "1.1.0",
+            "reference": "6.6"
+        }
+    ],
+    "references": [
+        "https://cloud.google.com/sql/docs/mysql/configure-private-ip",
+        "https://cloud.google.com/sql/docs/mysql/private-ip",
+        "https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints",
+        "https://console.cloud.google.com/iam-admin/orgpolicies/sql-restrictPublicIp"
+    ],
+    "dashboard_name": "Instances",
+    "path": "cloudsql.projects.id.instances.id",
+    "conditions": [
+        "and",
+        [
+            "cloudsql.projects.id.instances.id.public_ip",
+            "notEmpty",
```

Comment:
@SophieDorval this is being triggered when the value is `None`:

![sc_2021-04-12_12h35m21s](https://user-images.githubusercontent.com/4206926/114381604-98212300-9b8b-11eb-80ee-a5e11c8d93d7.png)